packages/ember-test-app/app/components/f/icon.gts

```diff
@@ -0,0 +1,112 @@
+import Component from '@glimmer/component';
+import Icon from '@nrg-ui/ember/components/icon';
+import FreestyleSection from 'ember-freestyle/components/freestyle-section';
+import FreestyleUsage from 'ember-freestyle/components/freestyle/usage';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { fn } from '@ember/helper';
+
+export default class extends Component {
+  colorOptions = [
+    'primary',
+    'secondary',
+    'success',
+    'danger',
+    'warning',
+    'info',
+    'light',
+    'dark',
+  ];
+
+  backgroundColorOptions = [
+    'Default',
```

Comment:
I implemented this Default option so users could quickly change the background color back to the default value, but I'm worried it could be misleading for implementation. I don't want users to think that 'Default' is a valid value to pass to backgroundColor. Will discuss to determine the pros and cons